middleware/admin/verify_attestation.py

```diff
@@ -23,13 +23,14 @@
 import json
 import hashlib
 import secp256k1 as ec
+import re
 from .misc import info, head, AdminError
 from .utils import is_nonempty_hex_string
 from .certificate import HSMCertificate
 
 
-UI_MESSAGE_HEADER = b"HSM:UI:5.1"
-SIGNER_MESSAGE_HEADER = b"HSM:SIGNER:5.1"
+UI_MESSAGE_HEADER = b"HSM:UI:5.X"
+SIGNER_MESSAGE_HEADER = b"HSM:SIGNER:5.X"
```

Comment:
Are these unused now? Can we use them for the regular expressions instead? So that it's easier to update when we need to bump a version.

Reply:
And also to avoid duplication like we are introducing right now.

Reply:
Pushed a new commit addressing this concern, let me know how it looks.

The solution has one minor drawback though. I had to change the error messages on lines `136` and `180` since now we use the regex match to compute the header length. As a consequence, in the event of an error, the entire message will be printed to the user, not only the header. Not a major problem IMO, since we're not losing any information, but curious to hear your thoughts.

Reply:
I understand. Looking at it, I think the resulting error message is still meaningful to somebody trying to debug a problem, so we should be fine.